package.json

```diff
@@ -252,6 +241,7 @@
 		"gettext-parser": "^4.0.3",
 		"glob": "^7.1.6",
 		"globby": "^10.0.1",
+		"gzip-size": "^5.1.1",
```

Comment:
I got an `eslint` error because this missing dependency, not directly related to this PR.